localnode/docker-compose.yml

```diff
@@ -0,0 +1,262 @@
+      - "-c"
+      - "random_page_cost=1.0"
+    ports:
+      - "5432:5432"
+    environment:
+      POSTGRES_DB: "bfg"
+      POSTGRES_HOST_AUTH_METHOD: "trust"
+    volumes:
+      - psql_data:/var
+    networks:
+      localnode:
+
+  bfgd:
+    build:
+      dockerfile: "./docker/bfgd/Dockerfile"
+      context: "./.."
+    deploy:
+      restart_policy:
+        condition: "any"
+    depends_on:
+      - "bfgd-postgres"
+      - "electrs"
+    ports:
+      ### CHANGE HOST PORTS
+      - "8082:8080"
+      - "8385:8383"
+    environment:
+      BFG_POSTGRES_URI: "postgres://postgres@bfgd-postgres:5432/bfg?sslmode=disable"
+      ### XXXX Make this variable and set to recent block
+      BFG_BTC_START_HEIGHT: "3195980"
+      BFG_EXBTC_ADDRESS: "electrs:50001"
+      BFG_LOG_LEVEL: "INFO"
+      BFG_PUBLIC_ADDRESS: ":8383"
+      BFG_PRIVATE_ADDRESS: ":8080"
+    networks:
+      localnode:
+
+  bssd:
+    build:
+      dockerfile: "./docker/bssd/Dockerfile"
+      context: "./.."
+    depends_on:
+      - "bfgd"
+    ports:
+      - "8081:8081"
+    environment:
+      BSS_BFG_URL: "ws://bfgd:8080/v1/ws/private"
+      BSS_LOG_LEVEL: "INFO"
+      BSS_ADDRESS: ":8081"
+    networks:
+      localnode:
+
+  prysm:
+    image: gcr.io/prysmaticlabs/prysm/beacon-chain:stable
+    command:
+      - "--execution-endpoint=http://geth-l1:8551"
+      - "--sepolia"
+      - "--jwt-secret=/tmp/jwt/jwt.hex"
+      - "--checkpoint-sync-url=https://sepolia.beaconstate.info"
+      - "--genesis-beacon-api-url=https://sepolia.beaconstate.info"
+      - "--accept-terms-of-use"
+      - "--datadir=/data/prysm"
+      - "--rpc-host=0.0.0.0"
+      - "--enable-debug-rpc-endpoints"
+      - "--grpc-gateway-host=0.0.0.0"
+      - "--enable-experimental-backfill"
+    volumes:
+      - prysm_data:/data
+      - "./jwt.txt:/tmp/jwt/jwt.hex:ro"
+    ports:
+      - 4000:4000
+    networks:
+      localnode:
+
+  geth-l1:
+    image: "ethereum/client-go@sha256:343cff592e1a9daf99a89261d3263941ff9c3350b62b3ebc0f7c30d35721d905"
+    ports:
+      - "8545:8545"
+    working_dir: "/tmp/geth"
+    command:
+      - "--sepolia"
+      - "--http"
+      - "--http.port=8545"
+      - "--http.addr=0.0.0.0"
+      - "--http.api=eth,net,engine,admin"
+      - "--authrpc.vhosts=*"
+      - "--authrpc.addr=0.0.0.0"
+      - "--authrpc.jwtsecret=/tmp/jwt/jwt.hex"
+      - "--http.vhosts"
+      - "*"
+      - "--datadir"
+      - "/tmp/geth"
+      - "--gpo.percentile=0"
+    volumes:
+      - geth_data:/tmp/geth
+      - "./jwt.txt:/tmp/jwt/jwt.hex"
+    networks:
+      localnode:
+
+  op-geth-l2-init:
+    image: alpine:latest
+    command:
+      - chown
+      - -R
+      - "65532" # geth user
+      - /tmp/datadir
+    volumes:
+      - op-geth_data:/tmp/datadir
+  op-geth-l2-init-tbc:
+    image: alpine:latest
+    command:
+      - chown
+      - -R
+      - "65532" # geth user
+      - /tbcdata
+    volumes:
+      - tbc_data:/tbcdata
+
+  op-geth-l2:
```

Comment:
annoyingly, it seems like we will have to p2p op-geth-l2 as well, let's add the toml file to this PR tomorrow